DependenciesGui/DependencyWindow.xaml.cs

```diff
@@ -399,8 +399,19 @@ private void ProcessPe(Dictionary<string, ImportContext> NewTreeContexts, PE new
 				{
 					ImportModule.PeProperties = ResolvedModule.Item2;
 
-					if (ResolvedModule.Item2 != null)
-						ImportModule.PeFilePath = ResolvedModule.Item2.Filepath;
+                    if (ResolvedModule.Item2 != null)
+                    {
+                        ImportModule.PeFilePath = ResolvedModule.Item2.Filepath;
+                        foreach (var Import in BinaryCache.LookupImports(DllImport, ImportModule.PeFilePath))
+                        {
+                            if (!Import.Item2)
+                            {
+                                ImportModule.Flags |= ModuleFlag.MissingImports;
+                                break;
+                            }
+
+                        }
+                    }
```

Comment:
This is indented with spaces,
some of the code was indented with spaces, some with tabs (I was not sure what is preferred)...

Reply:
Yeah Visual Studio's tab management is pretty horrid, and does not do well with other editors.
Since I was the only one commiting on this project that wasn't an issue, but I should probably normalize it in the near future :p

Reply:
You might wanna look into adding an https://editorconfig.org/ file in your repo.
VS 2015 needs an extension, but I think 2017 has builtin support?